frontend/src/components/common/ReleaseNotes/UpdatePopup.tsx

```diff
@@ -1,11 +1,22 @@
 import { Button, Snackbar } from '@material-ui/core';
 import React from 'react';
+import { useEffect, useState } from 'react';
 import { useTranslation } from 'react-i18next';
 
-function UpdatePopup(props: { releaseDownloadURL: string }) {
-  const [show, setShow] = React.useState(true);
+export interface UpdatePopupProps {
+  releaseDownloadURL: string;
+  open: boolean;
+}
+
+function UpdatePopup(props: UpdatePopupProps) {
   const { releaseDownloadURL } = props;
   const { t } = useTranslation('frequent');
+  const { open } = props;
```

Comment:
The open should just be joint with the releaseDownloadURL on line 12 is what I meant:
```typescript
const { releaseDownloadURL, open } = props;
```